src/packages/scalapb/update-hash.py

```
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+import json
+import subprocess
+import urllib.request
+import re
+import sys
+
+def get_latest_release():
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)